spyder/widgets/reporterror.py

```diff
@@ -21,11 +21,14 @@
 
 # Local imports
 from spyder import (__project_url__, __trouble_url__, dependencies,
-                    get_versions)
-from spyder.config.base import _, running_under_pytest
+                    get_versions_text)
+from spyder.config.base import _, is_pynsist, running_in_mac_app
 from spyder.config.gui import get_font
+from spyder.config.manager import CONF
```

Comment:
Please remove `CONF` from here. Instead, you need to inherit the `SpyderConfigurationAccessor` mixin in `SpyderErrorDialog` and then use its `get_conf` method to get configuration options.